src/proposer/proposer_rpc.cpp

```diff
@@ -4,7 +4,10 @@
 
 #include <proposer/proposer_rpc.h>
 
+#include <chainparams.h>
 #include <core_io.h>
+#include <injector.h>
```

Comment:
What is this include needed for?

The `injector.h` header file should be included, ideally, nowhere except in `init.cpp` as it in turn includes everything and the kitchen sink. Since components are injected they already have access to the components which are injected and no access to the Injector itself, or GetComponent, should be required elsewhere - except in code which is not bundled as a component, which is all the bitcoin legacy.

Reply:
fair enough.

Reply:
fair enough, I didn't realize that the RPC had components injected already.